src/components/primitives/Image/index.tsx

```diff
@@ -31,49 +31,87 @@ const StyledImage = styled(RNImage)<IImageProps>(
   customLayout
 );
 
-const Image = ({ source, ...props }: IImageProps, ref: any) => {
+const Image = (
+  { source, src, fallbackElement, ...props }: IImageProps,
```

Comment:
source, src and fallbackElement should be pass through usePropsResolution otherwise passing these props in baseStyle / default won't work.

Reply:
Fixed.